consumer/src/main/kotlin/streams/procedures/StreamsSinkProcedures.kt

```diff
@@ -1,21 +1,34 @@
 package streams.procedures
 
+import org.apache.commons.lang3.exception.ExceptionUtils
+import org.neo4j.graphdb.GraphDatabaseService
+import org.neo4j.kernel.internal.GraphDatabaseAPI
 import org.neo4j.logging.Log
 import org.neo4j.procedure.*
 import streams.StreamsEventConsumer
 import streams.StreamsEventConsumerFactory
+import streams.StreamsEventSink
 import streams.StreamsEventSinkConfigMapper
 import streams.StreamsSinkConfiguration
 import streams.config.StreamsConfig
+import streams.events.StreamsPluginStatus
+import streams.extensions.toPointCase
+import streams.serialization.JSONUtils
+import streams.utils.Neo4jUtils
+import java.util.stream.Collectors
 import java.util.stream.Stream
 
 class StreamResult(@JvmField val event: Map<String, *>)
+class StreamsSinkDTO(@JvmField val name: String, @JvmField val value: Any?)
```

Comment:
that's a really odd name as it leaks implementation details into the API

Reply:
changed